usage/rest-server/src/test/java/brooklyn/rest/BrooklynRestApiLauncher.java

```diff
@@ -150,6 +152,11 @@ public BrooklynRestApiLauncher withoutJsgui() {
         this.deployJsgui = false;
         return this;
     }
+    
+    public BrooklynRestApiLauncher disableHighAvailability(boolean value) {
```

Comment:
I agree with you, I also prefer positive naming. In this case though it's more specific. When the flag is `disableHighAvailability=true` the code calls
```
 mgmt.getHighAvailabilityManager().disabled();
```


Having it named `enableHighAvailability=true` I would expect that it calls start on the HA. By adding the flag I wanted to be able to say "don't disable HA".

So wdyt, I don't mind changing it, but `enableHighAvailability` could lead to unmet expectations?